services/tasks-svc/stories/TaskTemplateCRUD_test.go

```diff
@@ -126,4 +120,25 @@ func TestCreateUpdateGetTaskTemplate(t *testing.T) {
 	assert.Equal(t, t.Name()+" ST 2", template.Subtasks[0].Name)
 	assert.Equal(t, updateStRes.TaskTemplateConsistency, template.Consistency)
 
+	//
+	// create another template
+	//
+	createReq = &pb.CreateTaskTemplateRequest{
+		WardId:      nil,
+		Description: hwutil.PtrTo("Some Description"),
+		Subtasks:    make([]*pb.CreateTaskTemplateRequest_SubTask, 0),
+		Name:        t.Name() + " tt",
+	}
+	_, err = taskTemplateClient.CreateTaskTemplate(ctx, createReq)
+	assert.NoError(t, err, "could not create task template")
+	hwtesting.WaitForProjectionsToSettle()
+
+	//
+	// get all templates
+	//
+
+	templates, err := taskTemplateClient.GetAllTaskTemplates(ctx, &pb.GetAllTaskTemplatesRequest{})
```

Comment:
is there anything scoping the task templates response here (e.g., user, orga, ...)? if not this is racy against other tests, which might create task templates